owncloud/owncloud.py

```diff
@@ -878,6 +919,69 @@ def remove_user_from_group(self, user_name, group_name):
 
         raise ResponseError(res)
 
+    def add_user_to_subadminGroup(self, user_name, group_name):
+        """Adds a user to a subadmin group.
+
+        :param user_name:  name of user to be added to subadmin group
+        :param group_name:  name of subadmin group
+        :returns: True if user added
+        :raises: ResponseError in case an HTTP error status was returned
+
+        """
+
+        res = self.__make_ocs_request(
+            'POST',
+            self.OCS_SERVICE_CLOUD,
+            'users/' + user_name + '/subadmins',
```

Comment:
@tomneedham do you remember whether the provisioning API worked with encoded group names ?